<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Stop All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAllBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectAllBtn">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>400</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadBtn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="courseNo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>9</number>
    </property>
    <property name="placeholderText">
     <string>Course Number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reportBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Report a problem</string>
    </property>
   </widget>
   <widget class="QLabel" name="courseNameLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>450</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Course Name</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="chapterTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>680</width>
      <height>340</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>450</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Chapter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Download progress</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Chapter 1</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>chapter 2</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>chapter 3</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>chapter 4</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>chapter 5</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>chapter 6</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
